File: mario-less/mario.c
// Printing bricks(#) of a right-aligned pyramid of height n
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    // Prompting user for height
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Printing the pyramid blocks(#)
    for (int i = 1; i <= n; i++)
    {
        for (int j = i; j < n; j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

def main():
    height = get_height()
    for i in range(1, height + 1):
        for j in range(i, height):
            print(" ", end="")
        for k in range(1, i + 1):
            print("#", end="")
        print()

def get_height():
    while True:
        n = get_int("Height: ")
        if n >= 1 and n <= 8:
            break
    return n

main()